BootloaderCorePkg/Tools/SblFwUpdate.py

```diff
@@ -0,0 +1,32 @@
+## @ SblFwUpdate.py
+#
+# Copyright (c) 2017-2020, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+import os
+import re
+import sys
+import shutil
+
+def Usage():
+    print "SblFwUpdate"
+    print "Usage:"
+    print "    SblFwUpdate  <Input Capsule File>"
+
+def Main():
+    #
+    # Parse the options and args
+    #
+    argc = len(sys.argv)
+    if argc == 2:
+        shutil.copy(sys.argv[1], '/boot/efi/FwuImage.bin')
+        os.system('echo 1 > /sys/bus/wmi/devices/44FADEB1-B204-40F2-8581-394BBDC1B651/firmware_update_request')

```

Comment:
you might want to check if this file exists before you do "echo 1".
Also better to read it back to ensure the value has actually been changed.  Provide status to end user if this trigger request is successful or not.

Reply:
Done